### PR TITLE
Mccalluc/derived from sample

### LINKED ISSUE #2002: "Derived from sample" persists on search page, even when filter is removed.
Visit a page like https://portal.hubmapconsortium.org/search?ancestor_ids[0]=9c13d044a1529cb1f2596bc0b0325dbb&entity_type[0]=Dataset, and then remove the `Ancestor ID: 9c13d044a1529cb1f2596bc0b0325dbb` filter. It disappears, and the search is unconstrained, but the note "Derived from sample HBM998.KNJQ.234" in still above the search input box.

### PR DESCRIPTION
**Pull request by mccalluc**
Fix #2002. Not a high priority, but investigating the bug drifted into fixing the bug.

### CODE REVIEW

File: context/app/static/js/components/Search/SearchWrapper.jsx [hunk 1]
```diff
@@ -49,9 +53,40 @@ function SearchWrapper(props) {
     };
   }, [searchkit, setSearchHitsCount]);
 
+  const [message, setMessage] = useState(null);
+  const { elasticsearchEndpoint, nexusToken } = useContext(AppContext);
+  searchkit.setQueryProcessor((query) => {
```
Comment (john-conroy) [inline context/app/static/js/components/Search/SearchWrapper.jsx:58]: It seems searchkit has a `QueryAccessor` is this more along the lines of what we should use?

File: context/app/static/js/components/Search/SearchWrapper.jsx [hunk 2]
```diff
@@ -49,9 +53,40 @@ function SearchWrapper(props) {
...
+    if (!musts) {
+      setMessage(null);
+      return query;
+    }
+    const labels = {
+      'ancestor_ids.keyword': 'Derived from',
+      'descendant_ids.keyword': 'Ancestor of',
+    };
+    const provMusts = musts.filter(
+      (must) => 'term' in must && Object.keys(labels).some((labelKey) => labelKey in must.term),
```
Comment (john-conroy) [inline context/app/static/js/components/Search/SearchWrapper.jsx:72]: Are there 'musts' we wouldn't want to display notes for?

File: context/app/static/js/components/Search/SearchWrapper.jsx [hunk 3]
```diff
@@ -49,9 +53,40 @@ function SearchWrapper(props) {
...
+    );
+    setMessage(
+      provMusts.map((must) => {
+        return Object.entries(must.term).map(([k, v]) => (
+          <LookupEntity uuid={v} elasticsearchEndpoint={elasticsearchEndpoint} nexusToken={nexusToken}>
```
Comment (john-conroy) [inline context/app/static/js/components/Search/SearchWrapper.jsx:77]: This is the only place we're using `LookupEntity` would one of our hooks fit instead?

File: context/app/static/js/components/Search/SearchWrapper.jsx (hunk 1 above)
Comment (mccalluc) [reply context/app/static/js/components/Search/SearchWrapper.jsx:58]: Will look into that. Thanks!

**Comment by mccalluc** [comment]
Issue was closed won't-fix, and this code would need to be redone with the search replacement in any case. Closing.

**Pull request closed without merging**